Nodejs/Product/Nodejs/Snippets/1033/Nodejs/forin.snippet

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>forin</Title>
+      <Description>Code Snippet for For...In Loop</Description>
+      <Shortcut>forin</Shortcut>
+      <Author>Microsoft Corporation</Author>
+      <SnippetTypes>
+        <SnippetType>SurroundsWithStatement</SnippetType>
+        <SnippetType>Expansion</SnippetType>
+      </SnippetTypes>
+    </Header>
+    <Snippet>
+      <Code Language="Nodejs">
+        <![CDATA[for (var i in o) {
+   $selected$$end$

```

Comment:
The spacing in many of the snippets is inconsistent - in this case, there are three spaces for indents, in other cases, there two or four.